tests-legacy/PrestaShopBundle/Utils/DatabaseCreator.php

```diff
@@ -56,8 +56,6 @@ public static function createTestDB()
             exit(1);
         }
 
-        $process = new Process(PHP_BINARY . ' bin/console prestashop:schema:update-without-foreign --env=test');
```

Comment:
FYI: not needed thanks to this PR and https://github.com/PrestaShop/PrestaShop/pull/22823/files#diff-8af95ba09ba6393852c6f09c373d0732892ebb40bf2dfd1e1f3e23bc6e769aacL328